app/screens/send-bitcoin-screen/send-bitcoin-confirmation-screen.tsx

```diff
@@ -262,301 +251,110 @@ const SendBitcoinConfirmationScreen = ({
-      errorsMessage = joinErrorsMessages(errors)
-    }
-
-    if (data?.onChainPaymentSend?.errors.length) {
-      errorsMessage = errorsMessage + ` ${data?.onChainPaymentSend?.errors[0].message}`
-    }
-    return { status: data?.onChainPaymentSend.status, errorsMessage }
-  }
+    return async () => {
+      try {
+        logPaymentAttempt({
+          paymentType: paymentDetail.paymentType,
+          sendingWallet: sendingWalletDescriptor.currency,
+        })
+        const { status, errorsMessage } = await sendPayment()
+        logPaymentResult({
+          paymentType: paymentDetail.paymentType,
+          paymentStatus: status,
+          sendingWallet: sendingWalletDescriptor.currency,
+        })
 
-  const transactionPaymentMutation = (): (() => Promise<{
-    status: Maybe<PaymentSendResult>
-    errorsMessage: string
-  }>) => {
-    switch (paymentType) {
-      case "intraledger":
-        return payerWalletDescriptor.currency === WalletCurrency.Usd
-          ? payIntraLedgerUsd
-          : payIntraLedger
-      case "lightning":
-        if (isNoAmountInvoice) {
-          return payerWalletDescriptor.currency === WalletCurrency.Usd
-            ? payLnNoAmountUsdInvoice
-            : payLnNoAmountInvoice
+        if (!errorsMessage && status === "SUCCESS") {
+          navigation.dispatch((state) => {
+            const routes = [{ name: "Primary" }, { name: "sendBitcoinSuccess" }]
+            return CommonActions.reset({
+              ...state,
+              routes,
+              index: routes.length - 1,
+            })
+          })
+          return
         }
-        return payLnInvoice
-      case "onchain":
-        return payOnChain
-      case "lnurl":
-        return payLnInvoice
-      default:
-        throw new Error("Unsupported payment type")
-    }
-  }
 
-  const sendPayment = async () => {
-    setStatus(Status.LOADING)
-    try {
-      logPaymentAttempt({
-        paymentType,
-        sendingWallet: payerWalletDescriptor.currency,
-      })
-      const paymentMutation = transactionPaymentMutation()
-      const { status, errorsMessage } = await paymentMutation()
-      logPaymentResult({
-        paymentType,
-        paymentStatus: status,
-        sendingWallet: payerWalletDescriptor.currency,
-      })
-      if (!errorsMessage && status === "SUCCESS") {
-        setStatus(Status.SUCCESS)
-        navigation.dispatch((state) => {
-          const routes = [{ name: "Primary" }, { name: "sendBitcoinSuccess" }]
-          return CommonActions.reset({
-            ...state,
-            routes,
-            index: routes.length - 1,
-          })
-        })
-        return
-      }
+        if (status === "ALREADY_PAID") {
+          setPaymentError("Invoice is already paid")
+          return
+        }
 
-      if (status === "ALREADY_PAID") {
-        setPaymentError("Invoice is already paid")
-        return
+        setPaymentError(errorsMessage || "Something went wrong")
```

Comment:
string need translation

Reply:
This PR doesnt change the UI, it only refactors the logic for constructing and sending the payments. I think it makes sense to address the untranslated strings when we revisit the UI aspect of this flow in the redesigned screens.